appid: b22421bd
  # /*
  # * rdn:           合成音频数字发音方式
  # * volume:        合成音频的音量
  # * pitch:         合成音频的音调
  # * speed:         合成音频对应的语速
  # * voice_name:    合成发音人
  # * sample_rate:   合成音频采样率
  # * text_encoding: 合成文本编码格式
  # *
  # */
rdn: 0      #0 数值优先,(数字发音优先级)
            #1 完全数值,
            #2 完全字符串,
            #3 字符串优先
volume: 50  #合成音频的音量,调整范围：[0,100],值越大音量越高
pitch : 50
speed: 50   #合成音频对应的语速,调整范围：[0,100],值越大速度越快
voice_name : xiaoyan
sample_rate : 16000 #目前只支持16000 8000,默认16000
            